samples/balloon/inspect_balloon_data.ipynb

```diff
@@ -49,13 +49,13 @@
     "\n",
     "# Import Mask RCNN\n",
     "sys.path.append(ROOT_DIR)\n",
-    "import utils\n",
-    "import visualize\n",
-    "from visualize import display_images\n",
-    "import model as modellib\n",
-    "from model import log\n",
+    "import mrcnn.utils\n",
```

Comment:
I believe this should be `from mrcnn import utils`, otherwise references to `utils` in the code would fail. The same applies to several other import statements in this file and others.